### PR TITLE
Remove anonymous submodules

### PR DESCRIPTION
**Pull request by lekcyjna123**
- Refactor of Schedulers to remove Module from theirs arguments. 
- Changed anonymous submodules to named

Depends on #317

### CODE REVIEW

File: coreblocks/transactions/core.py [hunk 1]
```diff
@@ -292,7 +294,8 @@ def elaborate(self, platform):
         m = Module()
 
         for cc in _graph_ccs(rgr):
-            self.cc_scheduler(method_map, m, cgr, cc, porder)
+            mm = self.cc_scheduler(method_map, cgr, cc, porder)
+            m.submodules += mm
```
Comment (tilk) [inline coreblocks/transactions/core.py:298]: Why the temporary variable?

Comment (lekcyjna123) [reply coreblocks/transactions/core.py]: removed

File: coreblocks/transactions/core.py [hunk 2]
```diff
@@ -292,7 +294,7 @@ def elaborate(self, platform):
         m = Module()
 
         for cc in _graph_ccs(rgr):
-            self.cc_scheduler(method_map, m, cgr, cc, porder)
+            m.submodules += self.cc_scheduler(method_map, cgr, cc, porder)
```
Comment (xThaid) [inline coreblocks/transactions/core.py:297]: nit: maybe we can name these submodules like `scheduler0` etc.

**Review by xThaid — APPROVED** [review]

File: coreblocks/transactions/core.py (hunk 2 above)
Comment (tilk) [reply coreblocks/transactions/core.py:297]: Maybe let's do like @Arusekk proposed during the meeting: put them all inside a single Module, and add that module with a name.

Comment (lekcyjna123) [reply coreblocks/transactions/core.py]: Fixed

**Review by tilk — APPROVED** [review]

**Pull request merged**